src/klein/_interfaces.py

```diff
@@ -7,18 +7,21 @@
 works, since mypy doesn't otherwise get along with Zope Interface.
 """
 
-from typing import Mapping, Optional
+from typing import Mapping, Optional, Union
 
 from zope.interface import Attribute, Interface
 
 
+KleinQueryValue = Union[str, int, float]
```

Comment:
I know this is just moving, but seeing it here illustrates to me that we really ought to have put in a little textual description, maybe a `@var`, explaining what it's actually for...